content/en/docs/concepts/security/overview.md

```diff
@@ -0,0 +1,151 @@
+Access to etcd | Access to etcd (the datastore of Kubernetes) should be limited to the masters only. Depending on your configuration you should also attempt to use etcd over TLS. More info can be found here: https://github.com/etcd-io/etcd/tree/master/Documentation#security
+etcd Encryption | Wherever possible it's a good practice to encrypt all drives at rest, but since etcd holds the state of the entire cluster (including Secrets) its disk should especially be encrypted at rest.
+
+## Cluster
+
+This section will provide links to more exploration for securing the aspects of
+workloads in Kubernetes. There are essentially two areas of concern for securing
+Kubernetes:
+
+* Securing the components that are configurable which _make up_ the cluster
+* Securing the components which run _in_ the cluster
+
+
+### Components _of_ the Cluster
+For more information on securing the components of the cluster, go [here](/docs/tasks/administer-cluster/securing-a-cluster).
+
+### Components _in_ the Cluster (your application)
+Depending on the attack surface of your application, you may want to focus on specific
+aspects of security. For example, if you are running a service (Service A) that is critical
+in a chain of other resources and a separate workload (Service B) which is
+vulnerable to a resource exhaustion attack, by not putting resource limits on
+Service B you run the risk of also compromising Service A. Below is a table of
+links of things to consider when securing workloads running in Kubernetes.
+
+Area of Concern for Workload Security | Recommendation |
+------------------------------ | ------------ |
+RBAC Authorization (Access to the Kubernetes API) | https://kubernetes.io/docs/reference/access-authn-authz/rbac/
+Authentication | https://kubernetes.io/docs/reference/access-authn-authz/controlling-access/
+Application secrets management (and encrypting them in etcd at rest) | https://kubernetes.io/docs/concepts/configuration/secret/ <br> https://kubernetes.io/docs/tasks/administer-cluster/encrypt-data/
+Pod Security Policies | https://kubernetes.io/docs/concepts/policy/pod-security-policy/
+Quality of Service (and Cluster resource management) | https://kubernetes.io/docs/tasks/configure-pod-container/quality-service-pod/
+Network Policies | https://kubernetes.io/docs/concepts/services-networking/network-policies/
+TLS For Kubernetes Ingress | https://kubernetes.io/docs/concepts/services-networking/ingress/#tls
+
+
+
+## Container
+
+The price of admission to a Kubernetes cluster is a container. Because of this,
```

Comment:
Can this be stated in a more universal way? I have no problems with it as is, as a native English speaker.

Reply:
I agree. A container is a fundamental component.

Reply:
Yes